src/main/java/org/corfudb/infrastructure/log/RollingLog.java

```diff
@@ -39,10 +36,17 @@
 public class RollingLog extends AbstractLocalLog {
 
     private Map<Long, FileHandle> channelMap;
+    private String pathSeparator;
 
     public RollingLog(long start, long end, String path, boolean sync) {
         super(start, end, path, sync);
         channelMap = new HashMap<>();
+        if (File.pathSeparator.equals(":")) {
+            pathSeparator = "/";     // OS X ... use slash, please
```

Comment:
This comment (which I assume tells me that this is special case handling for OS X) - doesn't really tell me what this code does.